cmd/dist/delete.go

```diff
@@ -1,34 +1,39 @@
 package main
 
 import (
-	contextpkg "context"
 	"fmt"
 
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+
+	contentapi "github.com/docker/containerd/api/services/content"
 	"github.com/docker/containerd/log"
 	digest "github.com/opencontainers/go-digest"
 	"github.com/urfave/cli"
 )
 
 var deleteCommand = cli.Command{
 	Name:      "delete",
-	Aliases:   []string{"del"},
+	Aliases:   []string{"delete", "del", "remove", "rm"},
```

Comment:
so many aliases, `rm` and `remove` are most standard...might require rewording descriptions though